aiida_nanotech_empa/workflows/cp2k/data/atomic_kinds.yml

```diff
@@ -138,323 +137,322 @@ basis_set:
-   Si: ALL
-   P:  ALL
-   S:  ALL
-   Cl: ALL
-   Ar: ALL
-   K:  ALL
-   Ca: ALL
-   Sc: ALL
-   Ti: ALL
-   V:  ALL
-   Cr: ALL
-   Mn: ALL
-   Fe: ALL
-   Co: ALL
-   Ni: ALL
-   Cu: ALL
-   Zn: ALL
-   Ga: ALL
-   Ge: ALL
-   As: ALL
-   Se: ALL
-   Br: ALL
-   Kr: ALL
-   Rb: ALL
-   Sr: ALL
-   Y:  ALL
-   Zr: ALL
-   Nb: ALL
-   Mo: ALL
-   Tc: ALL
-   Ru: ALL
-   Rh: ALL
-   Pd: ALL
-   Ag: ALL
-   Cd: ALL
-   In: ALL
-   Sn: ALL
-   Sb: ALL
-   Te: ALL
-   I:  ALL
-   Xe: ALL
-   Cs: ALL
-   Ba: ALL
-   La: ALL
-   Ce: ALL
-   Pr: ALL
-   Nd: ALL
-   Pm: ALL
-   Sm: ALL
-   Eu: ALL
-   Gd: ALL
-   Tb: ALL
-   Dy: ALL
-   Ho: ALL
-   Er: ALL
-   Tm: ALL
-   Yb: ALL
-   Lu: ALL
-   Hf: ALL
-   Ta: ALL
-   W:  ALL
-   Re: ALL
-   Os: ALL
-   Ir: ALL
-   Pt: ALL
-   Au: ALL
-   Hg: ALL
-   Tl: ALL
-   Pb: ALL
-   Bi: ALL
-   Po: ALL
-   At: ALL
-   Rn: ALL
+  H: ALL
+  He: ALL
+  Li: ALL
+  Be: ALL
+  B: ALL
+  C: ALL
+  N: ALL
+  O: ALL
+  F: ALL
+  Ne: ALL
+  Na: ALL
+  Mg: ALL
+  Al: ALL
+  Si: ALL
+  P: ALL
+  S: ALL
+  Cl: ALL
+  Ar: ALL
+  K: ALL
+  Ca: ALL
+  Sc: ALL
+  Ti: ALL
+  V: ALL
+  Cr: ALL
+  Mn: ALL
+  Fe: ALL
+  Co: ALL
+  Ni: ALL
+  Cu: ALL
+  Zn: ALL
+  Ga: ALL
+  Ge: ALL
+  As: ALL
+  Se: ALL
+  Br: ALL
+  Kr: ALL
+  Rb: ALL
+  Sr: ALL
+  Y: ALL
+  Zr: ALL
+  Nb: ALL
+  Mo: ALL
+  Tc: ALL
+  Ru: ALL
+  Rh: ALL
+  Pd: ALL
+  Ag: ALL
+  Cd: ALL
+  In: ALL
+  Sn: ALL
+  Sb: ALL
+  Te: ALL
+  I: ALL
+  Xe: ALL
+  Cs: ALL
+  Ba: ALL
+  La: ALL
+  Ce: ALL
+  Pr: ALL
+  Nd: ALL
+  Pm: ALL
+  Sm: ALL
+  Eu: ALL
+  Gd: ALL
+  Tb: ALL
+  Dy: ALL
+  Ho: ALL
+  Er: ALL
+  Tm: ALL
+  Yb: ALL
+  Lu: ALL
+  Hf: ALL
+  Ta: ALL
+  W: ALL
+  Re: ALL
+  Os: ALL
+  Ir: ALL
+  Pt: ALL
+  Au: ALL
+  Hg: ALL
+  Tl: ALL
+  Pb: ALL
+  Bi: ALL
+  Po: ALL
+  At: ALL
+  Rn: ALL
 
 cutoff:
-   H:  600
-   He: 600
-   Li: 600
-   Be: 600
-   B:  600
-   C:  600
-   N:  600
-   O:  600
-   F:  600
-   Ne: 600
-   Na: 600
-   Mg: 600
-   Al: 600
-   Si: 600
-   P:  600
-   S:  600
-   Cl: 600
-   Ar: 600
-   K:  600
-   Ca: 600
-   Sc: 600
-   Ti: 600
-   V:  600
-   Cr: 600
-   Mn: 600
-   Fe: 1200
-   Co: 600
-   Ni: 600
-   Cu: 600
-   Zn: 600
-   Ga: 1200
-   Ge: 600
-   As: 600
-   Se: 600
-   Br: 600
-   Kr: 600
-   Rb: 600
-   Sr: 600
-   Y:  600
-   Zr: 600
-   Nb: 600
-   Mo: 600
-   Tc: 600
-   Ru: 600
-   Rh: 600
-   Pd: 1200
-   Ag: 600
-   Cd: 600
-   In: 600
-   Sn: 600
-   Sb: 600
-   Te: 600
-   I:  600
-   Xe: 600
-   Cs: 600
-   Ba: 600
-   La: 600
-   Ce: 600
-   Pr: 600
-   Nd: 600
-   Pm: 600
-   Sm: 600
-   Eu: 600
-   Gd: 600
-   Tb: 600
-   Dy: 600
-   Ho: 600
-   Er: 600
-   Tm: 600
-   Yb: 600
-   Lu: 600
-   Hf: 600
-   Ta: 600
-   W:  600
-   Re: 600
-   Os: 600
-   Ir: 600
-   Pt: 600
-   Au: 600
-   Hg: 600
-   Tl: 600
-   Pb: 600
-   Bi: 600
-   Po: 600
-   At: 600
-   Rn: 600
\ No newline at end of file
+  H: 600
+  He: 600
+  Li: 600
+  Be: 600
+  B: 600
+  C: 600
+  N: 600
+  O: 600
+  F: 600
+  Ne: 600
+  Na: 600
```

Comment:
Should we put Na: 1600 for cutoff ?

Reply:
no, 1600 would be neded in case of q=9, for teh moment we switch to q=1 and see how it works